projects/fsastorefrontlib/src/cms-components/my-account/quote/quotes/quotes.component.html

```diff
@@ -3,89 +3,179 @@ <h2 class="heading-headline py-3 mt-3 mb-5 text-center">
                   </div>
-                </ng-container>
-                <div class="info-card-data">
-                  <span class="label">{{
-                    'fscommon.expiryDate' | cxTranslate
-                  }}</span
-                  >:
-                  <span *ngIf="quote.expiryDate" class="value">{{
-                    quote.expiryDate | cxDate: 'dd MMM yyyy'
-                  }}</span>
-                  <span *ngIf="!quote.expiryDate" class="value">{{
-                    'fscommon.expiryNotConfirmed' | cxTranslate
-                  }}</span>
                 </div>
-                <div class="info-card-data">
-                  <span class="label">{{ 'fscommon.price' | cxTranslate }}</span
-                  >:
-                  <span class="value"
-                    >{{ quote.quotePrice }}
-                    <ng-container *ngIf="quote.paymentFrequency">
-                      / {{ quote.paymentFrequency }}</ng-container
-                    ></span
-                  >
+                <div class="info-card-image">
+                  <span class="image-wrapper text-center">
+                    <cx-media
+                      class="image-item"
+                      [container]="quote.defaultCategory.image"
+                    ></cx-media>
+                  </span>
                 </div>
               </div>
-              <div class="info-card-image">
-                <span class="image-wrapper text-center">
-                  <cx-media
-                    class="image-item"
-                    [container]="quote.defaultCategory.image"
-                  ></cx-media>
-                </span>
-              </div>
-            </div>
-            <ng-container
-              *ngIf="
-                !quote.isExpired &&
-                quote.quoteWorkflowStatus.code !== 'REFERRED'
-              "
-            >
-              <div class="info-card-links">
-                <a
-                  class="link"
-                  [routerLink]="
-                    {
-                      cxRoute: 'quoteDetails',
-                      params: { quoteId: quote.quoteId }
-                    } | cxUrl
-                  "
+              <ng-container
+                *ngIf="
+                  !quote.isExpired &&
+                  quote.quoteWorkflowStatus.code !== 'REFERRED'
+                "
+              >
+                <div
+                  class="info-card-links justify-content-between align-items-center"
                 >
-                  {{ 'fscommon.details' | cxTranslate }}
-                </a>
-                <a class="link" (click)="retrieveQuote(quote)">
-                  {{ 'fscommon.retrieve' | cxTranslate }}
-                </a>
-              </div>
-            </ng-container>
+                  <div
+                    class="checkbox-wrapper"
+                    [class.invisible]="
+                      quotesByCategory[quote.defaultCategory.name]?.length < 2
```

Comment:
If there is only one quote for a particular category element should be invisible.